test/forge/helpers/BaseTest.sol

```diff
@@ -52,8 +52,7 @@ contract BaseTest is Test {
     address internal MORPHO_OWNER = makeAddr("MorphoOwner");
     address internal MORPHO_FEE_RECIPIENT = makeAddr("MorphoFeeRecipient");
 
-    IMorpho internal morpho =
-        IMorpho(deployCode("lib/morpho-blue/out/Morpho.sol/Morpho.json", abi.encode(MORPHO_OWNER)));
+    IMorpho internal morpho = IMorpho(deployCode("out/Morpho.sol/Morpho.json", abi.encode(MORPHO_OWNER)));
```

Comment:
```suggestion
    IMorpho internal morpho = IMorpho(deployCode("Morpho", abi.encode(MORPHO_OWNER)));
```

should work too

Reply:
8d2a5b955f4225ff484c6ecd3b0ced562d8ce624